python/cudf/cudf/testing/io.py

```diff
@@ -76,3 +79,10 @@ def get_next_regression_params(self):
         self._idx += 1
         self._current_params = copy.copy(param)
         return dtypes_meta, num_rows, num_cols, seed
+
+    def get_rand_params(self, params):
+        params_dict = {}
+        for param, values in params.items():
+            params_dict[param] = np.random.choice(values)
```

Comment:
Can we use a dict comprehension here?

```suggestion
        params_dict = {param, np.random.choice(values) for param.values in params.items}
```